src/main/java/com/fortanix/sdkms/performance/sampler/AbstractSignatureSampler.java

```diff
@@ -43,6 +42,11 @@ public void setupTest(JavaSamplerContext context) {
         String keySize = context.getParameter(KEY_SIZE, "1024");
         String filePath = context.getParameter(FILE_PATH);
         int batchSize = context.getIntParameter(BATCH_SIZE, 0);
+        String hashAlorithm = context.getParameter(HASH_ALGORITHM);
+        DigestAlgorithm digestAlgorithm = DigestAlgorithm.fromValue(hashAlorithm.toUpperCase());
```

Comment:
There are 2 similar variable names: `hashAlgorithm` and `digestAlgorithm`, but different types. The variable names should have better meaning. I would suggest `inputDigestAlgorithm` and `digestAlgorithm`. Or better don't create the intermediate variable `hashAlgorithm` at all.

Reply:
fixed